### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KevinShimotakahara**
**SpringBoot Tickets**
https://www.notion.so/f3ba65c646a14d7f9b0174f6c7f2dc9f?v=373fb221b297417fb1f2fda5ef3ce86b&p=d7bbe8917f95474580fdf0330469b5e2

https://www.notion.so/f3ba65c646a14d7f9b0174f6c7f2dc9f?v=373fb221b297417fb1f2fda5ef3ce86b&p=9467b71389ba4143a590014e20cc17db

**Coding Challenges**
https://www.notion.so/Valid-Anagram-6e47efaa7bc545c7a93eedf9747fa39e
https://www.notion.so/Rotate-String-3e0daf3ea9a54672b11b0f63e262f649
https://www.notion.so/Valid-Parentheses-60305fe438f04e51884cf3cc1b98db6e
https://www.notion.so/String-to-Integer-atoi-437dcec88b4e41129e20221184ed5076

### CODE REVIEW

File: springboot/src/main/java/ca/jrvs/apps/trading/SwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        .select()
+        //Exclude error controller generated by spring http://bit.ly/2u0Duwd
+        .apis(Predicates.not(RequestHandlerSelectors.basePackage("org.springframework.boot")))
+        .paths(any())
+        .build()
+        .pathMapping("/")
+        .apiInfo(new ApiInfo("Jarvis Trading Rest App", "A Spring Rest API", "v1.0", null,
+            new Contact("Edward Wang", "jrvs.ca", "edward@jrvs.ca"), null, null,
+            new ArrayList<>()));
+  }
```
Comment (jarviscanada) [inline springboot/src/main/java/ca/jrvs/apps/trading/SwaggerConfig.java]: update this

File: springboot/src/main/java/ca/jrvs/apps/trading/controller/DashboardController.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    } catch (Exception e) {
+      throw ResponseExceptionUtil.getResponseStatusException(e);
+    }
+  }
+
+  @ApiOperation(value = "Show portfolio by trader ID")
+  @ApiResponses(value = {@ApiResponse(code = 404, message = "traderId is not found")})
+  @GetMapping(path = "/portfolio/traderId/{traderId}")
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
```
Comment (jarviscanada) [inline springboot/src/main/java/ca/jrvs/apps/trading/controller/DashboardController.java:50]: make sure you understand all annotations

File: springboot/src/main/java/ca/jrvs/apps/trading/dao/PositionDao.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+//    try {
+//      entity = Optional.ofNullable((T) getJdbcTemplate()
+//          .queryForObject(selectSql, BeanPropertyRowMapper.newInstance(getEntityClass()), id));
+//    } catch (IncorrectResultSizeDataAccessException e) {
+//      logger.debug("Can't find trader id:" + id, e);
+//    }
+//    return entity;
+//  }
+
+  /**
```
Comment (jarviscanada) [inline springboot/src/main/java/ca/jrvs/apps/trading/dao/PositionDao.java]: clean up unused code in all files

**Review by jarviscanada — APPROVED** [review]

File: springboot/src/main/java/ca/jrvs/apps/trading/SwaggerConfig.java (hunk 1 above)
Comment (KevinShimotakahara) [reply springboot/src/main/java/ca/jrvs/apps/trading/SwaggerConfig.java]: Done

**Pull request merged**